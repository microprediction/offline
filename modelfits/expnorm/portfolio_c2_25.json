{"g1": 0.8282353483553067, "g2": 0.2667655851093822, "logK": 0.9975995606194739, "loc": 0.08827451440449068, "logScale": -0.23346821710350096}