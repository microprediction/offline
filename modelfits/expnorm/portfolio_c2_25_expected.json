{"g1": 0.40980298108838586, "g2": 14.647626588058301, "logK": 0.862511795926663, "loc": -0.10058518702638194, "logScale": -1.8782302801301816}